common/workunit/workunit.cpp

```diff
@@ -2881,19 +2881,127 @@ class CDaliWorkUnitFactory : public CWorkUnitFactory, implements IDaliClientShut
                                                 ISecManager *secmgr,
                                                 ISecUser *secuser)
     {
+        class CQueryOrFilter : public CInterface, implements IInterface
+        {
+            int fmt;
+            StringAttr name;
+            StringArray values;
+        public:
+            IMPLEMENT_IINTERFACE;
+
+            CQueryOrFilter() {};
+
+            void setName(const char* _name) { name.set(_name); };
+            void setFormat(int _fmt) { fmt = _fmt; };
+            void appendValue(const char* _value) { values.append(_value); };
+            const char* getName() { return name.get(); };
+            int getFormat() { return fmt; };
+            StringArray& getValues() { return values; };
+        };
+        class CWorkUnitsPagerHelper : public CInterface, implements IInterface
+        {
+        public:
+            IMPLEMENT_IINTERFACE;
+
+            CWorkUnitsPagerHelper() {};
+            void setOrFilter(CQueryOrFilter& orFilter, int fmt, const char* name, const char* value, const char* sep)
+            {
+                if (!strieq(name, getEnumText(WUSFstate,workunitSortFields)) && !strieq(name, getEnumText(WUSFuser,workunitSortFields))
```

Comment:
style: split lines normally finish with an operator i.e. && on the end of the line